amd_ucode_info.py

```diff
@@ -228,6 +260,9 @@ def parse_ucode_file(opts, path, start_offset):
             if opts.extract:
                 extract_patch(opts, opts.extract, ucode_file, patch)
 
+            if opts.split:
```

Comment:
Having actions, such as extracting/splitting/merging, implemented as options (rather than separate commands, such as "amd_ucode_info {show|extract|split} ...") is a little unfortunate and makes the command line interface a bit suboptimal;  however, I wanted to preserve the existing interface and also didn't want to introduce additional logic dependencies between actions—each of them is considered and done independently, which seems like a cleaner interface to me ("there are -e and -s options" instead of "there are -e and -s options, they are mutually exclusive and -e has precedence" without any particular reasoning for such limitation).